Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [workspace]
```

Comment:
Besides the inclusion of more workspace dependencies, some items were re-ordered to make manual looks-up a bit easier